### ISSUE TITLE
Cannot find previous workflows

### ISSUE THREAD

**Issue by TKlingstrom** [SGBC/galaksio#26]
As soon as you close the Galaksio window the workflows becomes forgotten and impossible to retrieve (Workflow invocations menu remains empty).

**Comment by fikipollo** [comment]
@TKlingstrom I cannot reproduce this issue. When I close and reopen the window the information for the invocations remains there or is loaded again if was lost. However I saw that when starting a new session, the invocations menu was empty and showed a message "No invocations..." that was replaced seconds later by the retrieved invocations. I've added a "Loading" message that is showed while invocations are retrieved (see image). 

![screenshot_2017-09-06_09-58-56](https://user-images.githubusercontent.com/11427394/30100734-09c864a4-92ea-11e7-84ea-83c4bfc1af7d.png)
